### PR TITLE
feat: added support for query parameters in GET requests

### PR DESCRIPTION
**Pull request by alex-bradbury**
- Add query parameters option to GET request
- Change Ktor logger to only make debug logs

### CODE REVIEW

File: network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt [hunk 1]
```diff
@@ -46,7 +46,11 @@ class KtorHttpClient(
 
     @OptIn(ExperimentalSerializationApi::class)
     private fun makeHttpClient(userAgentGenerator: UserAgentGenerator): HttpClient {
-        val simpleLogger = Logger.SIMPLE
+        val simpleLogger = object : Logger {
+            override fun log(message: String) {
+                Log.d("GenericHttpClient", message)
```
Comment (flavius-mester) [inline network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt:51]:
```suggestion
                Log.d("KtorHttpClient", message)
```

**Review by flavius-mester — APPROVED** [review]

**Comment by flavius-mester** [comment]
on the failed build, have a look here, I think this might fix this issue as well

@Oliver-Binns  found the original problem

https://github.com/govuk-one-login/mobile-android-one-login-app/pull/171/files

File: network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt (hunk 1 above)
Comment (Oliver-Binns) [reply network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt]: Updated !

File: network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt [hunk 2]
```diff
@@ -165,6 +169,11 @@ class KtorHttpClient(
                         append(header.first, header.second)
                     }
                 }
+                url {
```
Comment (Oliver-Binns) [inline network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt:172]: @alex-bradbury I've rebased this on top of the changes, which has fixed the failing build.
Please can we get some unit tests for this functionality, then happy to get it merged 🙂

**Review by flavius-mester — APPROVED** [review]

**Pull request closed without merging**